planning/scenario_planning/common/motion_velocity_optimizer/package.xml

```diff
@@ -8,21 +8,21 @@
 
   <license>Apache 2.0</license>
 
-  <buildtool_depend>catkin</buildtool_depend>
-  <build_depend>roscpp</build_depend>
-  <build_depend>std_msgs</build_depend>
-  <build_export_depend>roscpp</build_export_depend>
-  <build_export_depend>std_msgs</build_export_depend>
-  <exec_depend>roscpp</exec_depend>
-  <exec_depend>std_msgs</exec_depend>
 
-  <depend>autoware_planning_msgs</depend>
+  <buildtool_depend>ament_cmake</buildtool_depend>
+
+  <export>
+    <build_type>ament_cmake</build_type>
+  </export>
+
+  <depend>rclcpp</depend>
   <depend>boost</depend>
   <depend>tf2</depend>
   <depend>tf2_ros</depend>
+  <depend>autoware_planning_msgs</depend>
+  <depend>autoware_debug_msgs</depend>
   <depend>geometry_msgs</depend>
   <depend>osqp_interface</depend>
+  <depend>eigen</depend>
```

Comment:
Modified here. https://github.com/tier4/Pilot.Auto/pull/77/commits/ec2bb12fe5ee7a67d964f47c56f158853c1e2300

It would be nice if you just look over it.